gtsfm/loader/argoverse_dataset_loader.py

```diff
@@ -0,0 +1,132 @@
+""" Loader that reads images from Argoverse directories on disk.
+
+Authors: John Lambert
+"""
+
+import glob
+import os
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+from gtsam import Cal3Bundler, Point3, Pose3, Rot3
+
+from argoverse.data_loading.simple_track_dataloader import SimpleArgoverseTrackingDataLoader
+from argoverse.utils.calibration import get_calibration_config
+from argoverse.utils.se3 import SE3
```

Comment:
should be rearranged with `argoverse` before `gtsam`.

Reply:
thanks, good catch